mkdocs.yml

```diff
@@ -42,7 +42,6 @@ nav:
       - Actions:
           - Overview: actions-intro.md
           - The project pipeline: actions-pipelines.md
-          - The cohortextractor action: actions-cohortextractor.md
```

Comment:
question: Is this page to be removed? Or is the intent just to remove it from the navigation alone (which is fine)?

Reply:
I'm going to salvage some possibly-useful cohortextractor stuff for some legacy docs.

Reply:
Great :+1: